neoload/commands/config.py

```diff
@@ -0,0 +1,31 @@
+import ast
+
+import click
+
+from neoload_cli_lib import config_global
+
+
+@click.command()
+@click.argument('command', type=click.Choice(['ls', 'set'], case_sensitive=False), required=True)
+@click.argument('key_values', required=False)
+def cli(command, key_values):
```

Comment:
We need a small documentation for this function like: "View and set the user-defined properties. It is stored in a local file."